Gemfile

```diff
@@ -67,7 +67,7 @@ end
 
 gem "rubyzip", '~> 1.2'
 
-gem 'rubocop', "~> 0.74"
+gem 'rubocop', '~> 0.74.0'
```

Comment:
Can we fix the failures instead and bump instead? It would be better to stay up to date?

Reply:
Most of the failures come from `.rubocop_rspec_base.yml`. Should I update RuboCop for all repos?
I can do that as a quick follow-up, but what do you think of merging this as is not to block other pull requests? 

```
$ rubocop/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for LineLength.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for MethodLength.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for SpaceAroundEqualsInParameterDefault.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for FileName.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for IfUnlessModifier.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop.yml: Warning: no department given for PerlBackrefs.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for AccessModifierIndentation.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for Alias.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for CaseEquality.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for CollectionMethods.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for CyclomaticComplexity.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for Documentation.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for DotPosition.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for DoubleNegation.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for EachWithObject.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for FormatString.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for HashSyntax.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for Lambda.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for LineLength.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for BinaryOperatorParameterName.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for PercentLiteralDelimiters.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for PredicateName.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for Proc.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for RescueException.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for SignalException.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for SpaceAroundEqualsInParameterDefault.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for StringLiterals.
/Users/pirj/source/rspec-dev/repos/rspec-rails/.rubocop_rspec_base.yml: Warning: no department given for TrivialAccessors.
Error: The `Layout/AlignParameters` cop has been renamed to `Layout/ParameterAlignment`.
(obsolete configuration found in .rubocop_rspec_base.yml, please update it)
The `Layout/IndentAssignment` cop has been renamed to `Layout/AssignmentIndentation`.
(obsolete configuration found in .rubocop_rspec_base.yml, please update it)
The `Layout/IndentFirstArgument` cop has been renamed to `Layout/FirstArgumentIndentation`.
(obsolete configuration found in .rubocop_rspec_base.yml, please update it)
The `Layout/IndentFirstArrayElement` cop has been renamed to `Layout/FirstArrayElementIndentation`.
(obsolete configuration found in .rubocop_rspec_base.yml, please update it)
The `Layout/IndentHeredoc` cop has been renamed to `Layout/HeredocIndentation`.
(obsolete configuration found in .rubocop_rspec_base.yml, please update it)
```

Reply:
I think rspec-rails is already divorced from the rest on this, given that its now independant from the others I think its ok to set its rubocop to newer on its own

Reply:
Awesome!

Updated to latest 0.77.0.
Merged `.rubocop_rspec_base.yml` into `.rubocop.yml`.
Extracted those cops' configs that were marked as "Over time we'd like to get this down" to `.rubocop_todo.yml`.